src/md/src/GettingStarted/Integrations.md

```diff
@@ -172,35 +170,35 @@ If you need to remove Jira integration, please perform actions:
 
 4. Click on a tab with existing integration
 
-5. Click on "Remove integration"
+5. Click on "Remove integration" link
 
 6. Submit an action
 
 [ ![Remove Global Jira Integration](Images/userGuide/gettingStarted/JiraGlobal.png) ](https://youtu.be/mEWP4txdJFA)
 
 
-### Project JIRA integration
+### Project JIRA Server integration
 
-#### Add new project Jira integration
+#### Add new project Jira Server integration
 
-If each project needs different Jira configurations, you need to unlink a project from Global configurations and add a project configuration. It means that now when new global integration will be added to the ReportPortal, it would not apply to the unlinked project. 
+If each project needs different Jira Server configurations, you need to unlink a project from Global configurations and add a project configuration. It means that now when new global integration will be added to the ReportPortal, it would not apply to the unlinked project. 
```

Comment:
If any project...
...you should unlink...
...when a new global integration is added... it won't be applied to